app/uk/gov/hmrc/pillar2/models/hods/subscription/common/SubscriptionResponse.scala

```diff
@@ -70,3 +70,12 @@ case class SecondaryContactDetails(
 object SecondaryContactDetails {
   implicit val format: OFormat[SecondaryContactDetails] = Json.format[SecondaryContactDetails]
 }
+
+case class DashboardInfo(
+  organisationName: String,
+  registrationDate: LocalDate
```

Comment:
Not sure LocalDate will work? From ETMP data comes with time. Please check the specs

Reply:
There is nothing in the spec or epid that is specific except for this :                 
registrationDate: 
                  allOf:
                    - $ref: '#/components/schemas/dateType'
                    - description: Registration date